src/index.js

```diff
@@ -1 +1,27 @@
-import './styles.css';
\ No newline at end of file
+import './styles.css';
+import { getScores, setScores } from './modules/api.js';
+
+const inputForm = document.querySelector('#input-form');
+const user = document.querySelector('#inputName');
+const score = document.querySelector('#inputScore');
+inputForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const obj = {};
+  obj.user = user.value;
+  obj.score = score.value;
+  setScores(obj);
+  inputForm.reset();
+});
+
+const render = async () => {
+  const list = await getScores();
+  const scoreList = document.querySelector('.score-list');
+  scoreList.innerHTML = '';
+  list.forEach((obj) => {
+    scoreList.innerHTML += `<li>${obj.user}: ${obj.score}</li>`;
```

Comment:
## [OPTIONAL]

- [ ] Kindly I would suggest you put padding around your text so that your text may be clear and more readable. 
![image](https://user-images.githubusercontent.com/65068771/153506545-d0517cff-138a-42a3-b32d-552990cad1d1.png). Check here :point_down: